packages/nx/src/daemon/server/project-graph-incremental-recomputation.ts

```diff
@@ -175,6 +176,8 @@ async function processCollectedUpdatedAndDeletedFiles() {
     );
     fileHasher.incrementalUpdate(updatedFiles, deletedFiles);
 
+    updateFilesInContext(updatedFiles, deletedFiles);
```

Comment:
It doesnt matter if its called before or after the file hasher update. But it definitely needs to be here because we need to know the file path and the new hash when we update the workspace context.